main.c

```diff
@@ -85,9 +85,15 @@
 #define UARTMUXSEL_SET_MIKROBUS 0x08
 #define UARTMUXSEL_SET_MSP430 0x09
 #define UARTMUXSEL_GET 0x0C
+#define BCF_I2CSLAVE_GET 0x0D
```

Comment:
Please consider dropping the term 'slave' and replacing with 'peripheral' or 'target'.

Reply:
Sure, will fix that